test/integration/apiary.js

```diff
@@ -0,0 +1,199 @@
+// Tests CAP-38 endpoints against the mock API server:
+// https://ammmock.docs.apiary.io/
+
+// All endpoints from here are tested:
+// https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU/edit
+const MOCK_SERVER = "http://private-anon-a06e1b25a0-ammmock.apiary-mock.com";
+
+describe("tests the /liquidity_pools endpoint", function() {
+  const lpId = "0569b19c75d7ecadce50501fffad6fe8ba4652455df9e1cc96dc408141124dd5";
+
+  it("GET /", function(done) {
+    let server = new StellarSdk.Server(MOCK_SERVER, {allowHttp: true});
+
+    chai.request(MOCK_SERVER)
+      .get("/liquidity_pools")
+      .end(function(err, res) {
+        if (err != null) done(err);
+        expect(res.body).not.to.be.null;
+
+        server
+          .liquidityPools()
+          .call()
+          .then(resp => {
+            expect(resp.records).to.deep.equal(res.body._embedded.records);
+            done();
+          })
+          .catch(err => done(err));
+      });
+  });
+
+  it('GET /<id>', function(done) {
```

Comment:
nit: just to make the code quicker to grasp when glancing over the `it`.

```suggestion
  it('GET /<pool-id>', function(done) {
```

Reply:
I like it :+1: